### PR TITLE
Colours updated for HTML output.

### PR DESCRIPTION
**Pull request by jolkdarr**
CSS simpler, rendering cleaner, report clearer.

### CODE REVIEW

**Comment by iloveeclipse** [comment]
Can you please attach picture before/after? I also see thatyou have removed the link to the project - why?

**Comment by jolkdarr** [comment]
The link is dropped in my personal version of the file. => **will be restored**.
ASAP, I will provide two HTMP reports that show off the improvement (according to me ;) ).

**Comment by jolkdarr** [comment]
Hello
I join two reports for the Javolution library:
[report.new.html.gz](https://github.com/spotbugs/spotbugs/files/1364542/report.new.html.gz)
[report.old.html.gz](https://github.com/spotbugs/spotbugs/files/1364543/report.old.html.gz)
Any suggestion is welcome.

File: spotbugs/src/xsl/color.xsl [hunk 1]
```diff
@@ -141,10 +142,11 @@
 	<body>
 
 	<h1>SpotBugs Report</h1>
-		<p>Produced using <a href="https://spotbugs.github.io">SpotBugs </a> <xsl:value-of select="/BugCollection/@version"/>.</p>
```
Comment (iloveeclipse) [inline spotbugs/src/xsl/color.xsl]: Please update your commit and restore the link.

**Pull request merged**